functions/tab.xfce4_terminal.fish

```diff
@@ -0,0 +1,3 @@
+function tab.xfce4_terminal -a cdto
+  xfce4-terminal --tab --working-directory "$cdto"
```

Comment:
it looks like `xfce4-terminal` includes an `-e` flag for executing a command inside the new terminal. does that execute the command and end the session, or does the window stick around after?

if it doesn't automatically close the session, let's add support for the second `$cmd` argument as well.